Sources/Components/TextField/TextField.swift

```diff
@@ -131,7 +131,7 @@ public class TextField: UIView {
         textField.enablesReturnKeyAutomatically = true
         textField.addTarget(self, action: #selector(textFieldDidChange), for: .editingChanged)
 
-        if FinniversKit.isDynamicTypeEnabled {
+        if !TestCheck.isTesting {
```

Comment:
i saw circle ci failing, i guess it's because of this... but what does change when running the snapshot tests regarding the content size category? :O